src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -0,0 +1,159 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 58 to help you plan.
+
+{# wf_updated_on: 2017-03-16 #}
+{# wf_published_on: 2017-03-16 #}
+{# wf_tags: deprecations,removals,chrome58 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 58 to help you plan. #}
+
+# Deprecations and Removals in Chrome 58 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 58,
+which is in beta as of March 16. This list is subject to change at any time.
+
+
+## Deprecate RTCRtcpMuxPolicy of "negotiate"
+
+The `rtcpMuxPolicy` is used by Chrome to specify its preferred policy regarding use of RTP/RTCP multiplexing. In Chrome 57, we changed the default `rtcpMuxPolicy` to "require" and deprecated "negotiate" for following reasons:
+
+* Non-muxed RTCP uses extra network resources.
+* Removing "negotiate" will make the API surface simpler, since an "RtpSender"/"RtpReceiver" will then only ever have a single transport.
+
+In Chrome 58, "negotiate" is deprecated. We believe this is a non-breaking change since the user will get a deprecation message and the `webkitRTCPeerConnection` can still be created successfully.
```

Comment:
Oh; it's not removed though. Since we merged the deprecation notice to M57, there shouldn't be any change from M57 to M58. The deprecation notice itself says it will be removed no earlier than M60.